### ISSUE TITLE
queue.pl exit with status 139

### ISSUE THREAD

**Issue by rishabh-09** [espnet/espnet#352]
Hello everyone! I was trying to run the toolkit on librispeech/chime5 dataset without any changes in codes. I am using queue.pl.
While running asrtrain.py stage fro librispeech, I get the floowing error in my log file:
![s1](https://user-images.githubusercontent.com/42341068/44025616-390d7cb4-9f0e-11e8-9514-1b9314faab96.png)

Also, same error comes when I run asrtrain.py stage on chime5.
I could not find the reason.
Also in train.log file in q folder, I get following error:
![s2](https://user-images.githubusercontent.com/42341068/44025754-93f2e0d8-9f0e-11e8-8869-4770cff219bc.png)

Also, I am submitting jobs as follows:
qsub  -V -cwd -o op.txt -e er.txt -S /bin/bash run.sh --ngpu 1

If someone knows about this issue, kindly help with the same. Thanks

**Comment by rishabh-09** [comment]
Following is the status of my log file with verbose=1.

![s3](https://user-images.githubusercontent.com/42341068/44051342-38ea5cfe-9f57-11e8-97ad-0b60efc055eb.png)

I could see the loss and gradient calculated, but there seems to be some problem with updating the parameters.

**Comment by sw005320** [comment]
I have no idea what happened, but it may be due to some grid engine setup. We actually have to hard code some GPU handling in https://github.com/espnet/espnet/blob/master/src/bin/asr_train.py#L170-L182, and you may also need such things.

**Comment by rishabh-09** [comment]
Thanks for your reply. I hardcoded for cuda vsible device as you suggested in my code.
But, still the error remains same. I tried with run.pl and there also same error appeared.
I am using pytorch 0.4.0 .. should i consider updating it ? or could there be some other dependency missing?
Kindly help. @sw005320

**Comment by sw005320** [comment]
can you try pytorch 0.4.1?

**Comment by rishabh-09** [comment]
sure .. i will try it..

**Comment by rishabh-09** [comment]
Hey @sw005320 I installed pytorch 0.4.1 and rebuilt pytorch-bindings....
But while running asrtrain.py , I am getting following error:
![error](https://user-images.githubusercontent.com/42341068/44213524-85ac7c80-a18b-11e8-8d8d-80bdcb496717.png)

I just wanted to know if toolkit has been fixed to pytorch  0.4.1?

Thanks

**Comment by kan-bayashi** [comment]
This is warpctc version mismatch issue.
Could you test using current master? It is better to start from setup.
